install-node.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
From what I can tell, https://github.com/mozilla/OpenWPM/blob/master/install-dev.sh isn't needed anymore. `jpm` is the build tool for the old add-on sdk extensions, and we install node separately in this file. If you agree, do you mind to remove `install-dev.sh` and `requirements-dev.txt` as part of this PR? The latter only contains `autopep8`, which someone can install if they wish -- it's not used by anything in the repo.

Reply:
This is correct, they can be removed.